src/AdminMenus/FormSettingsAdminSubMenu.php

```diff
@@ -141,7 +140,7 @@ protected function getMenuSlug(): string
 	 */
 	protected function getParentMenu(): string
 	{
-		return '';
+		return 'null';
```

Comment:
Why is this changed to string null? If the string should be empty by default, then better use `''`.

Reply:
Ok, but I think the empty string is a better choice than `'null'` string.